packages/server-wallet/src/db/migrations/20210312082700_remove_ledger_proposals.ts

```diff
@@ -0,0 +1,7 @@
+import {
+  up as createLedgerProposalsTable,
+  down as dropLedgerProposalsTable,
+} from './20201201095501_ledger_proposals';
+
+export const up = dropLedgerProposalsTable;
+export const down = createLedgerProposalsTable;
```

Comment:
It occurs to me here that there is an opportunity to rewrite history so that this ledger proposals table never existed. What are your thoughts on doing that?

Reply:
I think that it would be nice to collapse all migrations into a single, initial migration since we are no longer on the hook for maintaining/migrating databases that are in use. Filed https://github.com/statechannels/statechannels/issues/3379